.eslintrc.js

```diff
@@ -15,7 +15,19 @@ module.exports = {
         sourceType: "module",
         ecmaVersion: 2021,
       },
-      rules: {},
+      rules: {
+        "no-restricted-imports": [
+          "error",
+          {
+            patterns: [
+              {
+                group: [".*"],
+                message: "please use absolute import path instead.",
```

Comment:
Hey, I'm curious about this one.
Is this config for warning the dev about the absolute import if he/she doesn't follow the rule? Thanks.